tensorboard/webapp/widgets/data_table/data_table_component.scss

```diff
@@ -24,12 +24,23 @@ limitations under the License.
     position: sticky;
     text-align: left;
     top: 0;
+    vertical-align: bottom;
 
     @include tb-dark-theme {
       background-color: map-get($tb-dark-background, background);
     }
   }
 
+  .cell {
```

Comment:
Can we rename this to header or datatable-header?
Technically every cell in the table is a cell, but you only want this applied to the th header

Reply:
ok I saw it in runData section